main/RFXtrx.h

```diff
@@ -470,6 +470,8 @@ SDK version 4.9
 #define sTypeHEU 0x1
 #define sTypeANSLUT 0x2
 #define sTypeKambrook 0x03
+#define	sTypeSFSP_M 0x04
```

Comment:
Sorry you can not make any changes to this file.
Can we not use a normal on/off switch we already have? (a lighting2 ?)

Reply:
Ok, it was to clearly distinguish the origin of the command, because in multibloc we can have lot of blocks.
Maybe a over way to do that ? if not I still have the option to write the original reference in the default name when created...
![image](https://user-images.githubusercontent.com/12487713/31908197-654dba42-b836-11e7-9e43-0d9d6585f33b.png)

Reply:
You can use the 'SendSwitch' command for this ?